demisto_sdk/commands/common/tools.py

```diff
@@ -1,11 +1,13 @@
+from __future__ import annotations
```

Comment:
why?

Reply:
It allows using `|` for parameters typing like [here](https://github.com/demisto/demisto-sdk/pull/3709/files#diff-19219bb189e9e2c49968f53fb5cad06444a41c9502d62d4dade43e864a87cf59R693) on Python versions that are 3.7+ (without it, it's only allowed on 3.10+).